redaxo/src/addons/structure/pages/index.php

```diff
@@ -163,7 +163,7 @@
     $echo .= '  <tr>
                     <td class="rex-table-icon"><i class="rex-icon rex-icon-open-category"></i></td>
                     <td class="rex-table-id">-</td>
-                    <td data-title="' . rex_i18n::msg('header_category') . '"><a href="' . $context->getUrl(['category_id' => $category->getParentId()]) . '">..</a></td>
+                    <td data-title="' . rex_i18n::msg('header_category') . '"><a href="' . $context->getUrl(['category_id' => $category->getParentId()]) . '"><i class="fa fa-icon fa-icon-long-arrow-left"></i>' . rex_i18n::msg('category_parent') . '</a></td>
```

Comment:
Der Pfeil wird nicht angezeigt.
![image](https://user-images.githubusercontent.com/21292755/40885391-b7ad8652-6725-11e8-9029-8c0e0d0522dd.png)

Aus `fa-icon-long-arrow-left` müsste `fa-long-arrow-left` werden.
Und am besten etwas Abstand zwischen.
![image](https://user-images.githubusercontent.com/21292755/40885411-0656a81a-6726-11e8-986a-9b9bc4f7b1ca.png)

Reply:
wie wäre dieser Pfeil

Reply:
Und am besten nicht mit `fa-level-up-alt` benutzen sondern in die [_icon.scss](https://github.com/redaxo/redaxo/blob/master/redaxo/src/addons/be_style/scss/_icon.scss) als `rex-`-Klasse mit aufnehmen